src/siteconfig/templates/siteconfig/siteconfig_form.html

```diff
@@ -10,9 +10,7 @@
 
 <h3>Site Settings</h3>
 <form method="post" enctype="multipart/form-data">{% csrf_token %}
-    <p><input type="submit" value="Update" class="btn btn-success"/></p>
-   {{ form|crispy }}
-    <input type="submit" value="Update" class="btn btn-success"/>
+    {% crispy form %}
```

Comment:
Does this change remove the Update button from the top of the form?  If so, please revert.  The simplified code is nice but it's hear to get the button on the top of the form as well.

Reply:
Nope, looks good!